client/my-sites/email/email-management/titan-control-panel-redirect/index.jsx

```diff
@@ -95,15 +96,20 @@ class TitanControlPanelRedirect extends React.Component {
 	}
 }
 
-export default connect( ( state, ownProps ) => {
-	const site = getSiteBySlug( state, ownProps.siteSlug );
-	const siteId = site?.ID;
-	return {
-		domain: getSelectedDomain( {
-			domains: getDomainsBySiteId( state, siteId ),
-			selectedDomainName: ownProps.domainName,
-			isSiteRedirect: false,
-		} ),
-		siteId,
-	};
-} )( localize( TitanControlPanelRedirect ) );
+export default connect(
+	( state, ownProps ) => {
+		const site = getSiteBySlug( state, ownProps.siteSlug );
+		const siteId = site?.ID;
+		return {
+			domain: getSelectedDomain( {
+				domains: getDomainsBySiteId( state, siteId ),
+				selectedDomainName: ownProps.domainName,
+				isSiteRedirect: false,
+			} ),
+			siteId,
+		};
+	},
+	{
+		errorNotice,
```

Comment:
Just passing by - it might be a good idea to add this to `propTypes`.

Reply:
I thought about this, but then wondered where I should stop: should I also include `translate` in the prop types?

Reply:
Ahah, I didn't notice that this one was missing. So yes, I would include everything 😃 .

Reply:
Done!